MekHQ/unittests/mekhq/util/ImagesTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * MekHQ - Copyright (C) 2018 - The MekHQ Team
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ */
+package mekhq.util;
+
+import java.awt.Color;
+import java.awt.image.BufferedImage;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.function.IntBinaryOperator;
+import java.util.stream.Stream;
+
+import javax.imageio.ImageIO;
+import javax.swing.BoxLayout;
+import javax.swing.ImageIcon;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.SwingConstants;
+
+import org.junit.Test;
+
+import mekhq.IconPackage;
+
+@SuppressWarnings({"nls","javadoc"})
+public class ImagesTest {
+
+    public static void main(String[] args) throws Exception {
```

Comment:
Not really a unit test, it opens this:

<img width="750" alt="camos" src="https://user-images.githubusercontent.com/471835/45562072-9531ea80-b849-11e8-959b-e7ab52ca7b63.png">